window_manager/options.js

```diff
@@ -382,27 +395,37 @@ function getJsonTextFromEditor(editor) {
 
 /**
  * Loads the config, and sets up editors.
- *
- * @return {Promise<void>}
  */
-function onPageLoad() {
-  actionsEditor= createJSONEditorForElement('actionsInput', [], schemaValidator.actions);
+async function onPageLoad() {
```

Comment:
I think no changes are required to the onPageLoad - the only problem was that onLoad the onDisplayChanged method was called.
OnPageLoad is already calling validateEverything which updates displays.

Reply:
reverted. 
The only issue here is  that if the actions fails validation, then the displays list will not be populated until this validation error is fixed. 

(this also happens with onDisplayChanged() as well)